introduction.md

```diff
@@ -0,0 +1 @@
+Another awesome course by CodeSchool? Yes, it really is, you guys did it again.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.